recipes/cookbooks/packages/recipes/default.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Cookbook Name:: packages
+# Recipe:: default
+#
+# Copyright 2015, OSLL <kirill.yudenok@gmail.com>
+#
+# All rights reserved - Do Not Redistribute
+#
+
+# TODO - move here all packages based on product (galera, maxscale)
+
+# install additional packages for all platform
+case node[:platform_family]
+  when "debian", "ubuntu"
+    # install packages for ubuntu
+  when "rhel", "fedora", "centos"
+    # install packages for rhel
+    if node[:platform] == "centos" and node["platform_version"].to_f >= 7.0
```

Comment:
is it relevant only to centos and only 7? I guess net-tools can be missing in any distribution